src/pages/Swap/index_2.tsx

```diff
@@ -0,0 +1,3 @@
+export default function Swap2() {
```

Comment:
This file is not necessary. We can remove it

Reply:
agree